lib/mosaic-routing/src/main/java/org/eclipse/mosaic/lib/routing/pt/PtRouting.java

```diff
@@ -136,7 +136,12 @@ public PtRoutingResponse findPtRoute(PtRoutingRequest request) {
         );
         // ghRequest.setBlockedRouteTypes(request.getRoutingParameters().excludedPtModes);//FIXME generalize this
         ghRequest.setEarliestDepartureTime(departureTime);
-        ghRequest.setWalkSpeedKmH(SpeedUtils.ms2kmh(request.getRoutingParameters().getWalkingSpeedMps()));
+
+        if (request.getRoutingParameters().getWalkingSpeedMps() == null) {
+            ghRequest.setWalkSpeedKmH(5);
```

Comment:
This is a magic number

Reply:
I think I will throw an exception if it is not set, since with the current code it will be never null here.